verify-merge.py

```diff
@@ -35,75 +106,54 @@ def verify():
     if is_verification_error:
         sys.stderr.write('ERROR: One or more signatures failed verification.\n')
         exit(1)
-
     print('All signatures verified correctly.\n')
-    print('Beginning checksum comparison...\n')
+
+def verify_gpg_sig(sig_file):
```

Comment:
Why are you defining as separate function for this? It seems to be used only once.

Reply:
Yes it is mostly just for readability for now. I plan to expand these to add more functionality in a future PR (see "further checks" in the description above)